### PR TITLE
(PDB-2227) Create man pages for PuppetDB CLI

### PR DESCRIPTION
**Pull request by ajroetker**
This commit makes man pages for the PuppetDB CLI.

### CODE REVIEW

File: man/puppet-db.pod [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  $ puppet-db export [options]
+      The export action will export a PuppetDB archive from PuppetDB. The
+      default location of this archive will be './pdb-export.tgz' relative to
+      wherever the command was run from. You can specify a different location to
+      export to using the '--outfile <path>' option. Additionally you can
+      specify what level of anonymization you want for your archive using the
+      '--anonymization <str>' flag, for more information about PuppetDB archive
+      anonymization, consult the documentation at:
+      [http://docs.puppetlabs.com/puppetdb/master/anonymization.html]
```
Comment (senior) [inline man/puppet-db.pod]: I think we might want to switch this link to this http://docs.puppetlabs.com/puppetdb/master/migrate.html#exporting-data-from-an-existing-puppetdb-database we'll also need to update this when we release 4.0.0

Comment (senior) [inline man/puppet-db.pod]: Maybe this and the anonymization docs for --anonymize

File: man/puppet-db.pod [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+puppet-db [options] <action> [arguments]
+
+=head1 DESCRIPTION
+
+The C<puppet-db> tool allows you to perform PuppetDB administrative tasks such
+as exporting and anonymizing a backup of your PuppetDB or importing a backup to
+a PuppetDB. To learn more about the archive format and these administrative
+tasks in general, consult our documentation at:
+[http://docs.puppetlabs.com/puppetdb/master/anonymization.html]
```
Comment (senior) [inline man/puppet-db.pod]: Sorry, my comment below should be up here

**Comment by senior** [comment]
This PR doesn't include documentation on our config file. I think the way this is usually handled is a separate "puppetdb.conf" manpage, is that right @rbrw?

**Comment by ajroetker** [comment]
@senior pushed up the config file page

**Comment by senior** [comment]
This was merged in via the command-line, for some reason this PR didn't see it. Closing it as it's already in.

**Pull request closed without merging**